src/js/modules/infragistics.ui.editors.js

```diff
@@ -6538,19 +6519,25 @@
 			this._promptCharsIndices = [];
 
 			if (this._editMode) {
-				newValue = this._parseValueByMask(newValue);
-				this._editorInput.val(newValue);
-				if (selection !== undefined) {
-					// Move the caret
-					this._setSelectionRange(this._editorInput[ 0 ], selection.start, selection.end);
-				}
+				this._insertEditMode(newValue, selection);

```

Comment:
If we change the call to _insertEditMode and _insertDisplayMode - we need to be sure we have implementation into the base widgets. If inside a text editor I enter edit mode and type in composition when I submit the input the composition.end handler will try to call those functions - and it will throw an exception.

Reply:
Ignore my comment